src/Microsoft.Health.Fhir.Stu3.Core/Features/Resources/Create/CreateResourceHandler.cs

```diff
@@ -48,7 +50,7 @@ public async Task<UpsertResourceResponse> Handle(CreateResourceRequest message,
 
             resource.VersionId = result.Wrapper.Version;
 
-            return new UpsertResourceResponse(new SaveOutcome(resource, SaveOutcomeType.Created));
+            return new UpsertResourceResponse(new SaveOutcome(resource.ToResourceElement(), SaveOutcomeType.Created));
```

Comment:
I think the FhirDataStore could just return ResourceElement instead of Resource right?

Reply:
Yes it should, I think this can be done in ab#69127